### PR TITLE
Updated docs with script for stores

### PR DESCRIPTION
**Pull request by Delphine**
Updated with minimal info. Could add more details later if needed

### CODE REVIEW

File: products/appstores/new_release_android.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ The actions to perform are:
 This document assumes that you have set up the system as explained in [this document](../../config/setup_l10ndrivers_vm.md), so aliases like `lang_update` are available, repositories are already cloned in `~/mozilla/repositories`, Atom is available and includes the syntax highlighter for LANG files.
 
 **IMPORTANT:** Remember to run `gitup` before you do anything. If you run the command in the middle of the work, don’t forget to go back to the branch (it will checkout `master` for all repositories).
+Then remember to run the `app/scripts/update_shipping_locales.py` script in the `~/mozilla/git/stores_l10n/` directory, as stores_l10n is used to keep track of which locales ship in each version of supported apps (Firefox for Android/iOS, Focus for Android/iOS). In fact as a general rule, you should periodically run this script - especially before starting to work on stores_l10n.
```
Comment (flodolo) [inline products/appstores/new_release_android.md]: I'm not sure if this is the right place: if you run the script at this point, you will end up with changed files before creating a branch, new files will be potentially added to the appstores repository, etc.

I would put it into a separate section with its own title, e.g. "Updating the list of supported locales".

Another thing to call out: If the script changes the list of locales, it's important to run `lang_update all 12 all` to add missing files in appstores.

**Review by flodolo — COMMENTED** [review]
Since the page is quite long, add a 

```
<!-- toc -->
```

After the main title, like in [this page](https://raw.githubusercontent.com/mozilla-l10n/documentation/569c8d3b45e4b59d9c0886e7db1a3dd3bc627ec6/products/review/signoffs.md). It will create a table of contents automatically in GitBook.

**Comment by flodolo** [comment]
@Delphine 
I've made a [few changes](https://github.com/mozilla-l10n/documentation/pull/122/commits/d823ddbadcfa76914b1f90e1803dd1014553d80f), also to fix checks failing. Can you take a look?

**Comment by Delphine** [comment]
Looks good! Thanks for catching that error Flod - couldn't tell if I was supposed to skip a line or not for the 
```
<!-- toc -->
```
🦄

**Review by flodolo — APPROVED** [review]

**Pull request merged**